js/Ass.js

```diff
@@ -0,0 +1,35 @@
+$(document).ready (function(){
+
+    var female1 = prompt("Please enter the name of a female (1 of 2)");
```

Comment:
good prompt for value that gets assigned to a variable - and then that variable becomes the text for appropriate IDs in the html! Nice!